src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2114,7 +2118,7 @@
         "stacks": {
           "description": "An array of 'stack' objects relevant to the result.",
           "type": "array",
-          "minItems": 0,
+          "minItems": 1,
```

Comment:
this s/be zero. this construct is no different in this regard than codeFlows or graphs. @lgolding

Reply:
+1. I don't remember this change from Harlene's spreadsheet -- is it possible this change belongs somewhere else? There's nothing special about "stacks".

---
In reply to: [276013428](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276013428) [](ancestors = 276013428)

Reply:
good point! Harlene, let's double-check the spec to make sure you didn't flag something in external property files or something.

---
In reply to: [276015729](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276015729) [](ancestors = 276015729,276013428)

Reply:
marked it as a spec issue in excel, will revert it!

---
In reply to: [276016620](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276016620) [](ancestors = 276016620,276015729,276013428)